packages/manager/modules/pci/src/projects/projects.service.js

```diff
@@ -19,6 +24,33 @@ export default class {
     });
   }
 
+  getProjects(filters = [], sort = 'description', sortOrder = 'asc') {
+    return this.$q
+      .all([
+        this.publicCloud.getProjects(filters, sort, sortOrder),
+        this.publicCloud.getServices([
+          {
+            field: 'route.path',
+            comparator: 'eq',
+            reference: '/cloud/project/{serviceName}',
+          },
+        ]),
+      ])
+      .then(([projects, services]) =>
+        map(projects, (project) => {
+          set(
+            project,
+            'service',
+            find(
+              services,
+              (service) => project.project_id === service.resource.name,
+            ),
+          );
```

Comment:
To remove lodash methods (and imports)
```suggestion
          return new Project({
              ...project,
              service: services.find((service) => project.project_id === service.resource.name),
          });
```

Reply:
This is done